beacon-chain/state/v1/state_trie_test.go

```diff
@@ -177,6 +177,24 @@ func TestBeaconState_HashTreeRoot(t *testing.T) {
 	}
 }
 
+func BenchmarkBeaconState(b *testing.B) {
+	testState, _ := util.DeterministicGenesisState(b, 16000)
+	pbState, err := v1.ProtobufBeaconState(testState.InnerStateUnsafe())
+	require.NoError(b, err)
+
+	b.Run("Vectorized SHA256", func(b *testing.B) {
+		st, err := v1.InitializeFromProto(pbState)
```

Comment:
In `InitializeFromProtoUnsafe` there are a lot of allocations and copies that are not being benchmarked in the usual situation. This would most probably be not timed in the vectorized benchmark.